sdks/go/examples/cookbook/combine/combine.go

```diff
@@ -63,11 +63,16 @@ type extractFn struct {
 	MinLength int `json:"min_length"`
 }
 
+// A global context for simplicity.
+var ctx = context.Background()
```

Comment:
Since this is a cookbook the correct way to get a context is to have it as the first argument (as per the go idiom) to the ProcessElement method. The beam runtime will automatically provide a context for the function during execution.